mod-orders-storage/schemas/order_template_category_collection.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of order template categories",
+  "type": "object",
+  "javaName": "OrderTemplateCategoryCollection",
+  "properties": {
+    "orderTemplateCategories": {
+      "description": "Collection of order template categories",
+      "type": "array",
+      "id": "orderTemplateCategories",
+      "items": {
+        "type": "object",
+        "$ref": "order_template_category.json"
+      }
+    },
+    "totalRecords": {
+      "description": "The number of objects contained in this collection",
```

Comment:
Description can be updated, it is number of objects in DB in general for incoming search criteria.
TotalRecords needed to build appropriate amount of paginated requests.
For our case do not think that we can have many catgories but can remain schema as it, just update description